addons/sms/static/tests/mock_modals/partner.js

```diff
@@ -0,0 +1,22 @@
+import { fields, models } from "@web/../tests/web_test_helpers";
+
+export class Partner extends models.Model {
+    message = fields.Char();
+    foo = fields.Char();
+    mobile = fields.Char();
+    partner_ids = fields.One2many( {relation: "partner" });
```

Comment:
No, need to create these fields manually. Just give `_name = {modelname} " and it will define all the fields automatically.